catalogue/webapp/components/WorksSearchResult/WorksSearchResult.styles.tsx

```diff
@@ -54,9 +54,12 @@ export const WorkInformation = styled(Space).attrs({
   color: ${props => props.theme.color('neutral.600')};
 `;
 
-export const WorkInformationItem = styled.span`
-  &:not(:first-child)::before {
-    content: ' | ';
+export const WorkInformationItem = styled.span``;
```

Comment:
If this is just empty we could just remove the styled component and use a normal <span>?